rclcpp_generic/CHANGELOG.rst

```diff
@@ -0,0 +1,8 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Changelog for package rclcpp_generic
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+
+0.1.0 (2020-11-10)
+------------------
+Split off from rosbag2 repo.
```

Comment:
Anything else I should put here? Crediting the original authors?

Reply:
Generally you don't include CHANGELOG in the first commit - you want to do a followup bloom release process after the code has been checked in